css/menu.css

```diff
@@ -91,30 +90,25 @@ nav {
   font-family: 'Times New Roman', Times, serif;
 }
 
-.topnav {
-  overflow: hidden;
-  background-color: green;
-  padding: 10px;
+dt {
+  font-size: 30px;
```

Comment:
is px required? did you test this on larger and smaller device sizes?

Reply:
It was not tested. I will have to create another ticket to address the px vs % in our css files.